### PR TITLE
Building with ghc  8.8

### PR DESCRIPTION
**Pull request by rezafarzin**
retrying the change list

### CODE REVIEW

**Comment by alunduil** [comment]
It looks like this needs to be rebased on develop and possibly squashed.

File: .travis.yml [hunk 1]
```diff
@@ -28,8 +28,10 @@ before_cache:
 
 matrix:
   include:
-    - compiler: "ghc-8.6.1"
+    - compiler: "ghc-8.8.3"
```
Comment (alunduil) [inline .travis.yml]: I'm surprised this was the only version added by haskell-ci, but I'll see what the version specifier in the cabal file looks like.

**Review by alunduil — CHANGES_REQUESTED** [review]
Ah, I see.  The easy answer is to add 8.8 to the tested GHC versions and then just generate .travis.yaml with haskell-ci.  You shouldn't have to edit the travis configuration by hand.

File: .travis.yml (hunk 1 above)
Comment (rezafarzin) [reply .travis.yml]: If I paid a little more attention to your comment. I wouldn't need to debug the .travis.yml by hand. :S

**Review by alunduil — DISMISSED** [review]
That looks much better and the tests look like they're passing so we know we're good to go.

**Review by alunduil — CHANGES_REQUESTED** [review]
Looking closer I still see this based on the wrong branch.  Please rebase on develop and squash your commits into one.  After that this should easily be ready to merge.  Thanks!

**Review by alunduil — APPROVED** [review]

**Pull request merged**